### PR TITLE
added e2e test that syncs 200 tx via gossip and IBLT decode

### PR DESCRIPTION
**Pull request by woutslakhorst**
It simulates a Gossip queue overflow of 100tx. The result is a state, transactionSet and transactionList message.

### CODE REVIEW

File: nuts-network/gossip-overflow/run-test.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+   curl -s -X POST http://localhost:11323/internal/vdr/v1/did >/dev/null
+done
+
+echo "------------------------------------"
+echo "Performing assertions..."
+echo "------------------------------------"
+sleep 15
+# Assert that node B received all transactions
+RESPONSE=$(curl -s http://localhost:21323/status/diagnostics)
+if echo $RESPONSE | grep -q "transaction_count: 200"; then
```
Comment (gerardsn) [inline nuts-network/gossip-overflow/run-test.sh:37]: Test fails for me. Should this match the line below from the diagnostics?

`[DAG] Number of transactions: 200`

**Review by gerardsn — CHANGES_REQUESTED** [review]

File: nuts-network/gossip-overflow/run-test.sh (hunk 1 above)
Comment (reinkrul) [reply nuts-network/gossip-overflow/run-test.sh:37]: That's an ancient format of the diagnostics, your images might be out of date?

File: nuts-network/gossip-overflow/run-test.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+for i in {1..200}
+do
+   curl -s -X POST http://localhost:11323/internal/vdr/v1/did >/dev/null
+done
+
+echo "------------------------------------"
+echo "Performing assertions..."
+echo "------------------------------------"
+sleep 15
```
Comment (reinkrul) [inline nuts-network/gossip-overflow/run-test.sh:34]: This now becomes a performance test that asserts it completes within 15 seconds (and it always blocks for 15 seconds, even when it's faster). We should probably check every 1 second or so in a loop and time-out after 15 seconds. You can probably copy most of `waitForDCService`.

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: nuts-network/gossip-overflow/run-test.sh (hunk 1 above)
Comment (gerardsn) [reply nuts-network/gossip-overflow/run-test.sh:37]: only by a couple of months

File: nuts-network/gossip-overflow/run-test.sh (hunk 2 above)
Comment (woutslakhorst) [reply nuts-network/gossip-overflow/run-test.sh]: used API to do get diagnostics, better to use docker compose exec?

**Review by reinkrul — APPROVED** [review]

File: nuts-network/gossip-overflow/run-test.sh (hunk 2 above)
Comment (reinkrul) [reply nuts-network/gossip-overflow/run-test.sh]: docker compose exec uses the same API, but then using the client. So potato-potato, I think?

**Pull request merged**